### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by kidde60**
### Blog app - Views
**On this project I:**

- implemented view pages.
- added some styles.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,15 @@
-<h1>List of user by given id</h1>
\ No newline at end of file
+<div class="user-page">
+  <%= render "shared/details", user: @user %>
+    <div class="user-bio">
+     <strong class="bio">Bio: </strong> <br>
+     <p><%= @user.bio %></p>
+    </div>
+    <div class="user-posts">
+    <% @user.posts.each do |post| %>
```
Comment (elisha2kyakpo1) [inline app/views/users/show.html.erb]: - [ ]  Kindly make use of the `most_recent_posts` callback function you created in your user model to render the 3 most recent posts for a user.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+  <div>
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+       <%= render "shared/posts_details", post: post %>
+        <div class="comments">
+        <% if post.comments.empty? %>
+          <p>Empty</p>
+        <% else %>
+            <% post.comments.each do |comment| %>
+             <p class="comment-body"><%= comment.author.name %>: <%= comment.text %></p>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb:12]: - [ ]  Let's use the `most_recent_comments` callback function in our post model to render the 5 most recent comments for a post.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @kidde60, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @kidde60, 

To highlight 👍🏽 
✔️ No linter errors
✔️ App runs smoothly

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**